back/courses/models.py

```diff
@@ -1 +1,21 @@
-# Create your models here.
```

Comment:
There is a lot of unused import, you have to put only what is necessary in the file. (It is possible that they have been added automatically by visual code autocomplete)